apps/frontend/src/components/navbar.tsx

```diff
@@ -0,0 +1,43 @@
+import Image from 'next/image';
+
+import styles from '@/app/globals.module.css';
+
+function Navbar() {
+  return (
+    <nav className={styles.navbar}>
+      <div className='flex'>
+        <div className={styles['logo-container']}>
+          <Image src='/logo.png' alt='Logo' width={50} height={50} title='by KirDev' />
+        </div>
+        <div className={styles['link']}>
+          <a href='/' title='Kezdőlap'>
+            ProgramSCH
+          </a>
+        </div>
+      </div>
+      <div className='flex'>
+        <Icon href='' src='/new.png' alt='Új' title='Új esemény' />
+        <Icon href='' src='/profile.png' alt='Profil' title='Profil' />
+      </div>
+    </nav>
+  );
+}
+
+export default Navbar;
+
+type IconProps = {
+  src: string;
+  href: string;
+  title: string;
+  alt: string;
+};
+
+function Icon(props: IconProps) {
```

Comment:
Could be in its own file, because it is likely that we will use Icons later as well. 
Only put more components in the same file, besides the main one, if that won't be used again and if it is quite small, like this Icon component